### ISSUE TITLE
ЧЗХ пропаганда нездорового образа жизни на тау?

### ISSUE THREAD

**Issue by Motzord** [TauCetiStation/TauCetiClassic#11054]
Все мы прекрасно знаем, что хоть сервер и 16+, но иногда к нам залетают неокрепшие умы, а имея систему настроений, мы вдалбливаем детям некоторые очень и очень плохие догматы:

а) Алкоголь повышает настроение 
б) Сигареты повышают настроение
в) Наркотики вызывают эйфорию

г) Качалочка из-за боли понижает настроение

Отчего за сотни тысяч минуток у космонавтика в психике формируется явный психологический стереотип о том что всякие сигареты-алкоголь и прочая круто, а вот качалочка, здоровый образ жизни и прочее не круто!

Надо короче сделать все по другому и тогда воласу дадут медаль за пропаганду ЗОЖ среди детей и подростков а таукита станет национальным достоянием.

**Comment by Motzord** [comment]
Можно продолжить дальше, все мы знаем, что лучший способ убрать голод, является жратва сахара, что тоже является не здоровой хуйней поэтому я считаю, это тоже надо как-то фиксить, чтобы космонавты знали что еда с кучей сахара это плохо.

**Comment by Mercurialaste** [comment]
> 

сахар уже понерфили

**Comment by LudwigVonChesterfield** [comment]
я согласен, надо качалке дать бафф муда. как минимум достаточно сильный чтобы перекрывать боль.

но чтобы у уже накаченных меньше бафф муда был, так как они не получают боль.

а ещё алкашка должна разрушать накачанные мышцы.

**Comment by WatlerJe** [comment]
Это же пост-модерн.

**Comment by LudwigVonChesterfield** [comment]
#10827 пропагандирует здоровое питание.
